OpenRA.Mods.Common/ServerTraits/MasterServerPinger.cs

```diff
@@ -116,5 +146,26 @@ public void PingMasterServer(S server)
 
 			a.BeginInvoke(null, null);
 		}
+
+		void UpdateLANGameBeacon(S server, int numPlayers, int numSlots, int numBots, int numSpectators, Manifest mod, bool passwordProtected)
+		{
+			var settings = server.Settings;
+			var lanGameYaml =
+@"Game:
```

Comment:
Can you please add a `// TODO: Serialize and send client names` here?
This is a stub of a feature that has only been partly implemented, but it would be good to acknowledge that this bit will need to be updated in the future.  There is no point in actually implementing it now, I expect the details of the serialization will need to change.